docs/docs/development/hardware-integration/encoders.md

```diff
@@ -0,0 +1,80 @@
+---
+
+EC11 encoder support can be added to your board or shield by adding the appropriate lines to your board/shield's configuration (.conf), device tree (.dtsi), overlay (.overlay), and keymap (.keymap) files.
+
+## Configuration File
+
+In your configuration file you will need to add the following lines so that the encoders can be enabled/disabled:
+
+```ini
+# Uncomment to enable encoder
+# CONFIG_EC11=y
+# CONFIG_EC11_TRIGGER_GLOBAL_THREAD=y
+```
+
+These should be commented by default for encoders that are optional/can be swapped with switches, but can be uncommented if encoders are part of the default design.
+
+:::note
+If building locally for split boards, you may need to add these lines to the specific half's configuration file as well as the combined configuration file.
+:::
+
+## Devicetree File
+
+In your device tree file you will need to add the following lines to define the encoder sensor:
+
+```dts
+    left_encoder: encoder_left {
+        compatible = "alps,ec11";
+        a-gpios = <PIN_A (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>;
+        b-gpios = <PIN_B (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>;
+        steps = <80>;
+        status = "disabled";
+    };
+```
+
+Here you need to replace `PIN_A` and `PIN_B` with the appropriate pins that your PCB utilizes for the encoder(s). See [shield overlays section above](#shield-overlays) on the appropriate node label and pin number to use for GPIOs.
+
+The `steps` property should corresponded to the documented pulses per rotation for the encoders used on the keyboard, typically found on the datasheet of the component. If users use different encoders when they build, the value can be overridden in their keymap.
+
+Add additional encoders as necessary by duplicating the above lines, replacing `left` with whatever you would like to call your encoder, and updating the pins. Note that support for peripheral (right) side sensors over BLE is still in progress.
+
+Once you have defined the encoder sensors, you will have to add them to the list of sensors:
+
+```dts
+    sensors: sensors {
+        compatible = "zmk,keymap-sensors";
+        sensors = <&left_encoder &right_encoder>;
+        triggers-per-rotation = <20>;
+    };
+```
+
+In this example, a left_encoder and right_encoder are both added. Additional encoders can be added with spaces separating each, and the order they are added here determines the order in which you define their behavior in your keymap.
+
+In addition, a default value for the number of times the sensors trigger the bound behavior per full rotation is set via the `triggers-per-rotation` property. See [Encoders Config](../../config/encoders.md#devicetree) for more details.
+
+## Overlay File
+
+Add the following lines to your overlay file(s) to enable the encoder:
+
+```dts
+&left_encoder {
+    status = "okay";
+};
+```
+
+:::note
+For split keyboards, make sure to add left hand encoders to the left .overlay file and right hand encoders to the right .overlay file.
+:::
```

Comment:
````suggestion
Add the following lines to the .dts/.overlay file that contains the encoder to enable it:

```dts
&left_encoder {
    status = "okay";
};
```

Make sure to add this to the .dts/.overlay file, rather than any shared (.dtsi) files.
````